.github/openapi-spec/spec.yaml

```
@@ -8,6 +8,666 @@ servers:
+    post:
+      operationId: api_customers_post
+      tags:
+        - Customer
+      responses:
+        '201':
+          description: 'Customer resource created'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Customer'
+          links: {  }
```

yamllint (1.35.1)

[error] too many spaces inside empty braces

(braces)